src/qsec-calculatorservice/Controllers/MathController.cs

```diff
@@ -1,19 +1,33 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text.Encodings.Web;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using qsec_calculatorservice.Models;
+using Newtonsoft.Json;
 
 namespace qsec_calculatorservice.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
     public class MathController : ControllerBase
-    {
-  
-        public double Substract(double v1, double v2)
-        {
-            return v1 - v2;
-        }
+    {
+        [HttpPost]
+        [Route("Add")]
+        public async Task<IActionResult> Add([FromBody] CalcItem postModel)
+        {
+            if (postModel != null)
+                return Content(HtmlEncoder.Default.Encode($"{(int)(postModel.x + postModel.y)}"), "application/json");
```

Comment:
```suggestion
                return Ok(postModel.x + postModel.y);
```

Something like this should be nicer I think, remember casting to int means it won't work for values of type double.

Reply:
You do realize that this solution doesn't work according to the specs, like my original did? Now if you add two decimals, it doesn't round it to int like in the specs.

Reply:
Updated the specifications to support Finns.